CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ set( ECL_VERSION_MICRO git ) # with "new in Ert Version X.X.X"!
 # development version leading towards version MAJOR.MINOR.0
 
 execute_process(COMMAND date "+%Y-%m-%d %H:%M:%S" OUTPUT_VARIABLE RES_BUILD_TIME )
-string(STRIP ${RES_BUILD_TIME} RES_BUILD_TIME)
+string(STRIP "${RES_BUILD_TIME}" RES_BUILD_TIME)
```

Comment:
When you are here anyway - could you just please fix a bug `RES_BUILD_TIME` -> `ECL_BUILD_TIME`